Podcast/SeriesDetailViewController.swift

```diff
@@ -0,0 +1,162 @@
+        didSet {
+            updateViewData()
+        }
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        createSubviews()
+        // Do any additional setup after loading the view.
+        let s = Series()
+        s.title = "Dog Pods"
+        s.desc = "We talk lots about dogs and puppies and how cute they are and the different colors they come in and how fun they are."
+        s.largeArtworkImage = #imageLiteral(resourceName: "filler_image")
+        s.publisher = "Dog Lovers"
+        s.tags = ["Design", "Learning", "User Experience", "Technology", "Innovation", "Dogs"]
+        let episode = Episode(id: 0)
+        episode.title = "Puppies Galore"
+        episode.series = s
+        episode.dateCreated = Date()
+        episode.smallArtworkImage = #imageLiteral(resourceName: "filler_image")
+        episode.descriptionText = "Fun with dogs!"
+        episode.tags = ["Design", "Learning", "User Experience", "Technology", "Innovation", "Dogs"]
+        s.episodes = [episode]
+        series = s
+    }
+    
+    func createSubviews() {
+        let sframe = CGRect(x: 0, y:0, width: view.frame.width, height: seriesHeaderHeight)
+        seriesHeaderView = SeriesDetailHeaderView(frame: sframe)
+        seriesHeaderView.delegate = self
+        
+        let tframe = CGRect(x: 0, y: 0, width: view.frame.width, height: view.frame.height)
+        epsiodeTableView = UITableView(frame: tframe, style: .plain)
+        epsiodeTableView.delegate = self
+        epsiodeTableView.dataSource = self
+        epsiodeTableView.tableHeaderView = seriesHeaderView
+        epsiodeTableView.register(EpisodeTableViewCell.self, forCellReuseIdentifier: "EpisodeTableViewCellIdentifier")
+        epsiodeTableView.reloadData()
+        
+        view.addSubview(epsiodeTableView)
+    }
+    
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+    func updateViewData() {
+        guard let series = series else { return }
+        seriesHeaderView.updateViewWithSeries(series: series)
+        navigationItem.title = series.title
+        epsiodeTableView.reloadData()
+    }
+    
+    func subscribeButtonPressed(subscribed: Bool) {
+        
+    }
+    
+    func tagButtonPressed(index: Int) {
+        
+    }
+    
+    // MARK: - TableView
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        guard let series = series else { return 0 }
+        return series.episodes.count
+    }
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "EpisodeTableViewCellIdentifier") as! EpisodeTableViewCell
+        cell.delegate = self
+        guard let series = series else {
```

Comment:
If none of this works if the series doesn't get unwrapped, shouldn't the series not be optional?